csrc/device_lower/validation.cpp

```diff
@@ -198,13 +197,13 @@ namespace {
 // Check contiguity for all allocation domains associated with Misaligned
 // Vectorize ParallelType
 void checkContiguity(
-    const std::unordered_set<IterDomain*>& domains,
+    const std::unordered_set<IterDomain*>& dep_alloc_ids,
```

Comment:
Previously, `domains` include all IDs visited from the vectorized loop ID to the corresponding allocation IDs. Since only the allocation IDs are used here, `dep_alloc_ids` only includes the allocation IDs that are reached from the vectorized loop ID.